### PR TITLE
REM-668

### PR DESCRIPTION
**Pull request by Alinadimmokdad**
add fields for filtering 
add 2 total fields amount fc amount , 
change to correct endpoint

### CODE REVIEW

File: src/pages/post-outwards/index.js [hunk 1]
```diff
@@ -239,14 +348,31 @@ const Postoutwards = () => {
...
             paginationType='client'
             refetch={refetch}
             isLoading={false}
             maxAccess={access}
+            handleCheckboxChange={calcFc}
             showCheckboxColumn={true}
           />
         </Grow>
+        <Fixed>
+          <Grid container justifyContent='flex-end' spacing={2} sx={{ px: 2, mr: -11 }}>
```
Comment (RebeccaFrem02) [inline src/pages/post-outwards/index.js:361]: mr: -11? isn't there any better way to style this?

File: src/pages/post-outwards/index.js [hunk 2]
```diff
@@ -239,14 +348,31 @@ const Postoutwards = () => {
...
+            <Grid item xs={1.2}>
+              <CustomNumberField
+                name='totalFc'
+                label={_labels.totalFc}
+                value={formik.values.totalFc}
+                readOnly={true}
+                hidden={!(formik.values.countryId && formik.values.currencyId)}
+              />
+            </Grid>
+            <Grid item xs={1.2}>
```
Comment (RebeccaFrem02) [inline src/pages/post-outwards/index.js:371]: also here

File: src/pages/post-outwards/index.js [hunk 3]
```diff
@@ -145,49 +213,23 @@ const Postoutwards = () => {
                     values={formik.values}
                     valueField='recordId'
                     displayField={['reference', 'name', 'flName']}
-                    required
```
Comment (RebeccaFrem02) [inline src/pages/post-outwards/index.js:148]: why did you remove the required here if this field is required?

File: src/pages/post-outwards/index.js [hunk 4]
```diff
@@ -14,24 +13,41 @@ import { Fixed } from 'src/components/Shared/Layouts/Fixed'
...
       extension: RemittanceOutwardsRepository.Postoutwards.qry,
       parameters: `_countryId=${formik.values.countryId}&_currencyId=${formik.values.currencyId || 0}&_corId=${
         formik.values.corId || 0
-      }&_dispersalType=${formik.values.dispersalType || 0}`
+      }&_dispersalType=${formik.values.dispersalType || 0}&_fromAmount=${formik.values.fromAmount || 0}&_toAmount=${
+        formik.values.toAmount || 0
+      }&_fromDate=${formattedFromDate}&_todate=${formattedToDate}`
     })
       .then(response => {
         setData(response.list || [])
```
Comment (RebeccaFrem02) [inline src/pages/post-outwards/index.js:53]: remove catch here please

File: src/pages/post-outwards/index.js [hunk 5]
```diff
@@ -51,10 +67,15 @@ const Postoutwards = () => {
...
       corId: '',
-      dispersalType: ''
+      dispersalType: '',
+      totalFc: '',
+      totalAm: '',
+      fromDate: '',
+      toDate: ''
     },
     access,
     enableReinitialize: true,
```
Comment (RebeccaFrem02) [inline src/pages/post-outwards/index.js:81]: remove try catch in the submit function please

**Comment by RebeccaFrem02** [comment]
image
got this error

**Comment by RebeccaFrem02** [comment]
image
even after filling some fields so idk what is the problrm here

File: src/pages/post-outwards/index.js (hunk 3 above)
Comment (Alinadimmokdad) [reply src/pages/post-outwards/index.js:148]: not required any more

File: src/pages/post-outwards/index.js (hunk 1 above)
Comment (Alinadimmokdad) [reply src/pages/post-outwards/index.js:361]: i am not styling i am centering the total below the field

File: src/pages/post-outwards/index.js [hunk 6]
```diff
@@ -116,7 +152,39 @@ const Postoutwards = () => {
...
+    formik.setFieldValue('totalFc', 0)
+  }, [
+    formik.values.countryId,
+    formik.values.currencyId,
+    formik.values.corId,
+    formik.values.dispersalType,
+    formik.values.fromDate,
+    formik.values.toDate,
+    formik.values.fromAmount,
+    formik.values.toAmount
```
[the remark this answers is not in the document]
Comment (Alinadimmokdad) [reply src/pages/post-outwards/index.js:165]: no , the total Fc and total am are also in the formik values

File: src/pages/post-outwards/index.js (hunk 3 above)
Comment (RebeccaFrem02) [reply src/pages/post-outwards/index.js:148]: than please remove it from the validationSchema

File: src/pages/post-outwards/index.js [hunk 7]
```diff
@@ -239,14 +344,31 @@ const Postoutwards = () => {
...
             refetch={refetch}
             isLoading={false}
             maxAccess={access}
+            handleCheckboxChange={calcFc}
             showCheckboxColumn={true}
           />
         </Grow>
+        <Fixed>
+          <Grid container justifyContent='flex-end' spacing={2} sx={{ px: 2 }}>
+            <Grid item xs={1.6}>
```
Comment (RebeccaFrem02) [inline src/pages/post-outwards/index.js:358]: still using decimals can't we use it as 2 or find another solution?
